doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -80,7 +80,7 @@ pwsh /home/azure-sdk-for-net/eng/scripts/automation/Invoke-DataPlaneGenerateSDKP
 - `-service` takes Azure client service directory name. ie. purview. It equals to the name of the directory in the specification folder of the azure-rest-api-specs repo that contains the REST API definition file.
 - For `- namespace`, please use one of the pre-approved namespace groups on the [.NET Azure SDK Guidelines Approved Namespaces list](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list). This value will also provide the name for the shipped package, and should be of the form `Azure.<group>.<service>`.
 - `-sdkPath` takes the address of the root directory of sdk repo. e.g. /home/azure-sdk-for-net
-- `-inputfiles` takes the address of the Open API spec files,  separated by semicolon if there is more than one file.  The Open API spec file can be local file, e.g. ./swagger/compute.json, or the web address of the file in the `azure-rest-api-specs` repo.  When pointing to a file in the `azure-rest-api-specs` repo, make sure to include the commit id in the URI, i.e. `https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json`.  This ensures that you can choose the time to upgrade to new swagger file versions.
+- `-inputfiles` takes the address of the Open API spec files,  separated by semicolon if there is more than one file.  The Open API spec file can be local file, or the web address of the file in the `azure-rest-api-specs` repo.  When pointing to a local file, make sure to use **absolute path**, i.e. /home/swagger/compute.json. When pointing to a file in the `azure-rest-api-specs` repo, make sure to include the commit id in the URI, i.e. `https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json`. This ensures that you can choose the time to upgrade to new swagger file versions.
```

Comment:
If we support [Resolve-Path](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/resolve-path?msclkid=fb39864acf3811ec9d01f492e29b7525&view=powershell-7.2), then we should support relative path, right? Then here we don't need to emphasize `absolute path`, I guess.

Reply:
If you want to refer to an local swagger files as input-file, you need to add absolute path, because it will be inserted into autorest.md and will be used late, so it will be absolute path, otherwise, when it use `dotnet build /t:GenerateCode` it will fail. So I emphasize `absolute path` to ask them to provide absolute path. 
Yes, we can use Resolve-Path to translate relative path to absolute path, but since the -inputfiles are list of files which may be local file or web address, we need to write script to support this. Current we have not support yet, so ask them to provide absolute path.

Reply:
We may support in future